src/javascripts/Views/referencedByView.js

```diff
@@ -1,48 +1,36 @@
-ngapp.controller('referencedByViewController', function($scope, $element, $timeout, htmlHelpers, treeService, referencedByViewService, referencedByViewColumnsService, hotkeyService, nodeSelectionService, treeViewService, treeColumnService, contextMenuService, contextMenuFactory) {
+ngapp.controller('referencedByViewController', function($scope, $element, $timeout, htmlHelpers, gridService, referencedByViewService, columnsService, hotkeyService, nodeSelectionService, treeColumnService, contextMenuService, contextMenuFactory) {
     // link view to scope
     $scope.view = $scope.$parent.tab;
     $scope.view.scope = $scope;
 
-    $scope.allColumns = referencedByViewColumnsService.columns;
+    $scope.allColumns = columnsService.getColumnsForView('referencedByView');
     $scope.contextMenuItems = contextMenuFactory.referencedByViewItems;
 
     // inherited functions
-    treeService.buildFunctions($scope, $element);
-    treeViewService.buildFunctions($scope, $element);
+    gridService.buildFunctions($scope, $element);
     referencedByViewService.buildFunctions($scope);
     treeColumnService.buildFunctions($scope, '.referenced-by-view', true);
     nodeSelectionService.buildFunctions($scope, false);
     hotkeyService.buildOnKeyDown($scope, 'onTreeKeyDown', 'referencedByView');
 
     $scope.open = function(node) {
-        const record = node.handle ? node.handle : node;
-        $scope.record = record;
-        $scope.syncWithLinkedViews(record);
+        let recordView = $scope.view.linkedRecordView;
+        if (recordView) {
+            recordView.scope.record = xelib.GetElementEx(node.handle, '');
+        } else {
+            $scope.record = xelib.GetElementEx(node.handle, '');
```

Comment:
is this behavior we want to have?  navigating through references on the referenced by view by opening them?  it feels somewhat strange to me, but maybe it is sensible when it isn't linked to a record view?  :thinking:

Sure, seems fine.